app/src/main/java/org/simple/clinic/patient/PatientRepository.kt

```diff
@@ -783,6 +783,13 @@ class PatientRepository @Inject constructor(
     )
   }
 
+  fun updatePatientReassignmentEligibilityStatus(patientUuid: UUID) {
+    database.patientDao().updatePatientReassignmentEligibilityStatus(
+        patientUuid = patientUuid,
+        isEligibleForReassignment = isPatientEligibleForReassignment(patientUuid)
+    )
+  }
```

Comment:
What happens when a sync is performed, does the flag gets reset to false as we are not pushing it to server.

Reply:
It won't reset, since it's a local field and we use `upsert` function for updating existing patients during sync.

https://github.com/simpledotorg/simple-android/blob/49543a6e8c427ba6b6060ce176ff8ad18d586e4d/app/src/main/java/org/simple/clinic/patient/Patient.kt#L175